scarlet/component.py

```diff
@@ -281,8 +281,11 @@ def frame(self):
         return self._tree[0].frame
 
     @property
-    def nodes(self):
-        """Initial list that generates the tree.
+    def sources(self):
```

Comment:
Since you changed this, maybe the `n_nodes` property should be changed to `n_sources`